shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2369,8 +2369,12 @@ - (void)dismissKeyboardScreenshot {
 - (void)showKeyboardAndRemoveScreenshot {
   [UIView setAnimationsEnabled:NO];
   [_cachedFirstResponder becomeFirstResponder];
-  [UIView setAnimationsEnabled:YES];
-  [self dismissKeyboardScreenshot];
+  // UIKit does not immediately access the areAnimationsEnabled Boolean so a delay is needed before
+  // returned
+  dispatch_after(dispatch_time(DISPATCH_TIME_NOW, 0.1 * NSEC_PER_SEC), dispatch_get_main_queue(), ^{
```

Comment:
We should extract a constant and use that here:
```objc
// Detailed explanation of why this is necessary.
constexpr kKeyboardAnimationDelay = 0.1 * NSEC_PER_SEC;
```

Reply:
Definitely! Just added.